repos/docker-creds-provider.proj

```diff
@@ -0,0 +1,30 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <PackagesOutput>$(ProjectDirectory)/src/Valleysoft.DockerCredsProvider/bin/$(Configuration)/</PackagesOutput>
+    <RepoApiImplemented>false</RepoApiImplemented>
+    <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
+    <DockerCredsProviderPackageVersion>2.1.0</DockerCredsProviderPackageVersion>
+  </PropertyGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="RepoBuild">
+    <PropertyGroup>
+      <BuildCommandArgs>$(ProjectDirectory)/src/Valleysoft.DockerCredsProvider/Valleysoft.DockerCredsProvider.csproj</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:Configuration=$(Configuration)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /v:$(LogVerbosity)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) $(RedirectRepoOutputToLog)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:Version=$(DockerCredsProviderPackageVersion)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:TargetFrameworks=$(NetCurrent)</BuildCommandArgs>
+    </PropertyGroup>
+
+    <Exec Command="$(DotnetToolCommand) restore /bl:restore.binlog $(BuildCommandArgs)  " EnvironmentVariables="@(EnvironmentVariables)" WorkingDirectory="$(ProjectDirectory)" IgnoreStandardErrorWarningFormat="true" />
```

Comment:
Is this space intentional?  Seems inconsistent with the other commands.
```suggestion
    <Exec Command="$(DotnetToolCommand) restore /bl:restore.binlog $(BuildCommandArgs)" EnvironmentVariables="@(EnvironmentVariables)" WorkingDirectory="$(ProjectDirectory)" IgnoreStandardErrorWarningFormat="true" />
```

Reply:
I realize this was probably copied, put given the line wrapping I see in GH, consider formating so each element is on a separate line.